README.md

```diff
@@ -24,23 +24,24 @@ The board has 4 columns:
 This is what the board currently looks like:
 ![initial-board](public/initial-board.png)
 
-The final design of the board is:
-![final-board](public/final-board.png)
-
 ## How to Start
 
 This is a simple create-react-app application, the instructions on how to run can be found in [here](CONTRIBUTING.md).
-We are using pure React here with styling done in pure CSS, but feel free to approach this however you want. If you want to use libraries, we encourage you to - there is no right or wrong answer here! We use tailwind and styled-components for styling internally, but we don't expect you to be familiar with these up front.
+We are using pure ReactJS with TypeScript here with styling done in pure CSS, but feel free to approach this however you want. If you want to use libraries, we encourage you to - there is no right or wrong answer here! We use tailwind and styled-components for styling internally, but we don't expect you to be familiar with these up front.
 
 ## Tasks
 
-1. Please add validations to the member form, and relevant error and helper messages.
+1. Please add validations to the member form, relevant error and helper messages.
 2. Once 'Submit' is pressed, a card will be created on under _Unclaimed_ column.
 3. The card will contain all the details entered into the form.
 4. Please save new card data to _local storage_.
 5. Previous entered cards will reload onto the board as saved in _local storage_.
 6. Please add styling to the form and page.
```

Comment:
Might be worth taking page out here if we also mention it in bonus task 3

Reply:
I was thinking about that, but if they decide not to implement the bonus task then they might not see it